go/control/api/api.go

```diff
@@ -45,8 +45,9 @@ type NodeController interface {
 	// and shut down.
 	UpgradeBinary(ctx context.Context, descriptor *upgrade.Descriptor) error
 
-	// CancelUpgrade cancels a pending upgrade, unless it is already in progress.
-	CancelUpgrade(ctx context.Context) error
+	// CancelUpgrade cancels the specific pending upgrade, unless it is already in progress.
+	// TOOD: currently pending upgrade name uniqueness is not enforced.
+	CancelUpgrade(ctx context.Context, name string) error
```

Comment:
Make sure to flag this as a breaking gRPC API change.

Reply:
noted in 3536.breaking.md